### DISCUSSION TITLE
Trackpad and TrackPoint input is suuuuper wonky

### DISCUSSION THREAD

**Discussion by Trigex** [valnoxy/t440p-oc#69] [Q&A]
Hi! Big Sur is running great on my T440p thanks to this project, but mouse input is barely usable :( I have the t440p's stock trackpad, which is terrible as is, but it's almost unusable with this macOS configuration. The dedicated right click button on the trackpad just doesn't work, you have to press with 2 fingers (I'm a trackpoint user, so that's a huge inconvenience. And on top of that, it's a dice roll if the 2 finger press will even work, it seems like there's a extremely specific position to trigger it and hitting it is a pain. Middle click scrolling just doesn't work, which is also an issue as a trackpoint user. Neither can I disable the trackpad (The bios setting doesn't effect it), so when using the trackpoint the mouse is constantly sliding around every where because the trackpad is being skidded slightly.

Does anyone suggestions to fix these problems? I know replacing the trackpad with the T450's trackpad should help with skidding and stuff, but I don't think it would with the right and middle buttons functionality not being there. Are there some drivers or something to fix these problems? Thanks a ton!

**Comment by Trigex** [comment]
I realized the core of the issue was the trackpad buttons don't work at all, just the clickpad. Any solution?

**Reply by dayatnhbtc** [reply]
Hi. Have you fix this issues?

**Reply by Trigex** [reply]
Unfortunately not. I just use the clickpad now.

**Reply by dayatnhbtc** [reply]
Still using heckintosh? 
Do you have other problems?

**Reply by Trigex** [reply]
Yup, and I've got no other issues, except for subpar performance on Monterey, been meaning to downgrade to Big Sur.

**Reply by dayatnhbtc** [reply]
So this will be fine when you use mouse right? 
I should try this out
